api/api.proto

```diff
@@ -5628,7 +5628,7 @@ message SdkVersion {
     // SDK version major value of this specification
     Major = 0;
     // SDK version minor value of this specification
-    Minor = 173;
+    Minor = 174;
```

Comment:
if you are not changing any proto definitions, you don't need to bump up the SDK version. @lpabon Correct me if I am wrong.

Reply:
Thanks @adityadani, I too had seen few PRs where the version was not bumped up. Since we are not sure about it, cautioning on the safer side and bumping it up even though its not strictly required.